### ISSUE TITLE
docker run then always stop at "+ pulseaudio -D --verbose --exit-idle-time=-1 --disallow-exit" 

### ISSUE THREAD

**Issue by xianqiliu** [livekit/egress#480]
![image](https://github.com/livekit/egress/assets/74473717/a62262da-62cf-4cbb-9984-f100c9e20a06)

at the beginning, I run it locally on windows, so I know next there's should be displaying something as following:
```
I: [pulseaudio] main.c: Daemon startup successful.
+ exec egress
```

but when I run it locally on centos, it always wait/stop at ```+ pulseaudio -D --verbose --exit-idle-time=-1 --disallow-exit``` as the picture shown above, and thers's not any error showing, thus, I don't have any idea how to solve this problem.

I will be really appreciated if any one can help me.

**Comment by lcodecorex** [comment]
It occurred on Ubuntu 16.04 also.

**Comment by xianqiliu** [comment]
> It occurred on Ubuntu 16.04 also.

I add ```--security-opt seccomp=unconfined``` then it works in my side.

**Comment by heyarviind** [comment]
> > It occurred on Ubuntu 16.04 also.
> 
> @lcodecorex I add `--security-opt seccomp=unconfined` then it works in my side.

I added this but on instance restart I get the same error

**Comment by frostbyte73** [comment]
Should be fixed in latest (v1.7.12). This was related to recent chrome sandboxing/docker permission updates
